test/unit/math/mix/fun/to_complex_test.cpp

```diff
@@ -0,0 +1,8 @@
+#include <test/unit/math/test_ad.hpp>
+#include <complex>
+
+TEST(mathMixMatFun, to_complex) {
```

Comment:
This needs tests for both the nullary and binary calls.  I think there's a corresponding `expect_common_binary` that could be called with a 2-arg lambda.